staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -285,7 +285,30 @@ message LinuxSandboxSecurityContext {
     // * localhost/<full-path-to-profile>: the profile installed on the node.
     //   <full-path-to-profile> is the full path of the profile.
     // Default: "", which is identical with unconfined.
-    string seccomp_profile_path = 7;
+    string seccomp_profile_path = 7 [deprecated=true];
+    // Seccomp profile for the sandbox, defaults to the `RuntimeDefault`
```

Comment:
This comment needs to be updated as well. Thanks!

Reply:
Done :+1: